marlowe-playground-client/src/Simulation/BottomPanel.purs

```diff
@@ -97,9 +98,9 @@ bottomPanel state =
 
   errors = state ^. (_marloweState <<< _Head <<< _editorErrors)
 
-  hasRuntimeWarnings = state ^. (_marloweState <<< _Head <<< _transactionWarnings <<< to Array.null <<< to not)
+  hasRuntimeWarnings = maybe false (\x -> x ^. (_transactionWarnings <<< to Array.null <<< to not)) (state ^. (_marloweState <<< _Head <<< _executionState))
```

Comment:
another excellent use case case for `is`

Reply:
Sorry, I am not sure how we can use lenses to extract the `Slot` as a maybe using lenses